test/com/xilinx/rapidwright/edif/TestEDIFNetlist.java

```diff
@@ -0,0 +1,48 @@
+package com.xilinx.rapidwright.edif;
+
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import com.xilinx.rapidwright.design.Design;
+import com.xilinx.rapidwright.device.Part;
+import com.xilinx.rapidwright.device.PartNameTools;
+import com.xilinx.rapidwright.support.CheckOpenFiles;
+
+class TestEDIFNetlist {
+
+    private static final String PART_NAME = "xcvu440-flga2892-2-e";
```

Comment:
For runtime reasons consider a smaller device?

Reply:
Switched to xcku040.